.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  pull_request:
+    branches: [ "main" ]
+  issue_comment:
+    types: [ created ]
+
+name: Quarto Deploy Preview
+
+jobs:
+  delete-bot-comments:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-deleter
+        uses: maheshrayas/action-pr-comment-delete@v3.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          org: ${{ github.repository_owner }}
+          repo: ${{ github.event.repository.name }}
+          user: 'github-actions[bot]'
+          issue: ${{ github.event.number }}
+
+  build-deploy-preview:
+    # Deploy a preview only if
+    # - the event is a PR
+    # - the PR is not from a fork
+    if: |
+      (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork != true)
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
```

Comment:
@seankross so this setup requires a DESCRIPTION file in the repo - see error https://github.com/fhdsl/personas/actions/runs/9865033089/job/27241162325?pr=1#step:4:145 - you could add a valid desc file or remove this line in the yaml file

Reply:
@sckott I added a description file that I think is valid, still failing though.

Reply:
fixed, had to update netlify tokens, see link below